src/ontology/mondo-edit.obo

```diff
@@ -615764,6 +615766,15 @@ is_a: MONDO:0006552 {source="https://orcid.org/0000-0002-0819-0473"} ! folliculi
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0819-0473
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/pull/8094" xsd:anyURI
 
+[Term]
+id: MONDO:1060107
+name: CYP7B1-related disorder of oxysterol accumulation
+def: "Any disorder of oxysterol accumulation caused by biallelic loss of function variants in the CYP7B1 gene." [https://clinicalgenome.org/affiliation/40097/]
+synonym: "CYP7B1-related disorder of oxysterol accumulation" EXACT CLINGEN_LABEL [https://clinicalgenome.org/affiliation/40097/]
+is_a: MONDO:0002525 {source="https://clinicalgenome.org/affiliation/40097/"} ! inherited lipid metabolism disorder
```

Comment:
you can also add the axiom for the gene:
'has material basis in germline mutation in' some CYP7B1